drivers/tty/serdev/pibridge.c

```diff
@@ -407,7 +407,7 @@ int pibridge_req_io(u8 addr, u8 cmd, u8 *snd_buf, u16 snd_len, u8 *rcv_buf,
 	}
 	crc = pibridge_crc8(0, (u8 *) &pkthdr, sizeof(pkthdr));
 
-	to_receive = min((u16) pkthdr.len, rcv_len);
+	to_receive = min((u8) pkthdr.len, rcv_len);
```

Comment:
Nit: Unnecessary cast fom u8 to u8.

Reply:
Hm, why did you mark this resolved even though the code wasn't changed?

Reply:
Looks like I forgot to comment on this before I marked it resolved: we need this cast since unlike in struct   pibridge_pkthdr_gate where "len" is a u8, "len" is a bitfield in struct pibridge_pkthdr_io. Omitting the cast results in a compiler error.

Reply:
Got it, thanks for the explanation.